.appveyor.yml

```diff
@@ -0,0 +1,133 @@
+version: '{branch}-{build}'
+os: Visual Studio 2015
+clone_folder: C:\projects\oiio
+test: off
+configuration:
+    - Release
+    - Debug
+platform:
+    - x64
+# branches:
+#   only:
+#     - master
+environment:
+  matrix:
+    # - CMAKE_PLATFORM: "Visual Studio 14 2015"
+    #   PYTHON_DIR: "C:\\Python27"
+    - CMAKE_PLATFORM: "Visual Studio 14 2015 Win64"
+      PYTHON_DIR: "C:\\Python27-x64"
+      tbs_arch: "x64"
+      tbs_tools: "msvc14"
+      tbs_static_runtime: 0
+
+install:
+  - cinstall: python
+  - cinst ninja
+  - ps: if($env:build_platform -eq 'x64') {
+          $env:vcvar_arg = 'x86_amd64';
+        }
+        else {
+          $env:vcvar_arg = 'x86';
+        }
+  - call "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/vcvarsall.bat"
+  - set PATH=C:/Sys/bin;C:/tools/ninja;%PATH%
+  - set CMAKE_PREFIX_PATH=C:/Sys;%APPVEYOR_BUILD_FOLDER%/deps
+  - set CMAKE_LIB_PATH=C:/Sys/lib;%APPVEYOR_BUILD_FOLDER%/deps/lib
+  - set CMAKE_INCLUDE_PATH=C:/Sys/include;%APPVEYOR_BUILD_FOLDER%/deps/include
+  - mkdir c:\sys
+  - mkdir deps
+  - cd deps
+  - mkdir lib
+  - mkdir include
+
+  # ZLIB
+  # - svn co https://svn.blender.org/svnroot/bf-blender/trunk//lib/win64_vc14/zlib
+  # - set CMAKE_PREFIX_PATH=%CMAKE_PREFIX_PATH%;%APPVEYOR_BUILD_FOLDER%/deps/zlib
+  # FYI, alternate way to get zlib:
+  - appveyor DownloadFile https://github.com/madler/zlib/archive/v1.2.8.tar.gz
+  - 7z x v1.2.8.tar.gz -so | 7z x -si -ttar
+  - cd zlib-1.2.8
+  - cmake -G "Visual Studio 14 2015 Win64" .  -DCMAKE_INSTALL_PREFIX=C:/Sys -DBUILD_STATIC=%BUILD_STATIC% -G Ninja
```

Comment:
Seems a little strange that there's both `-G "Visual Studio 14 2015 Win64"` and `-G Ninja` here.